### PR TITLE
docs: update manifesto

### PR DESCRIPTION
**Pull request by achingbrain**
Refs #35

- Removes permissions section as it's not relevant to general usage
- Updates out of date sections and adds links to relevant modules
- Adds emojis

### CODE REVIEW

File: MANIFESTO.md [hunk 1]
```diff
@@ -23,27 +22,32 @@ For example:
 - Other applications may be deployed in extremely adversarial environments, and should limit the number of dependencies (throughout the dependency tree) to reduce the opportunities for supply chain attacks
 - The user should not have to include the code for features their application does not use
 
-The core of Helia will be very focused on use as a library: just [js-libp2p], a [blockstore], [js-bitswap] and a POSIX-like API which will be extendable to add additional features such as IPNS, an RPC-API, etc.
+The core of Helia will be very focused on use as a library: just [js-libp2p], a [datastore], and a [blockstore] that transparently uses [js-bitswap] to load any requested blocks not already in the blockstore from the network.
```
[the remark this answers is not in the document]
Comment (achingbrain) [reply MANIFESTO.md:25]: Right now we only have bitswap for transferring blocks, I guess we could look at how to use other protocols when something appears?

Bitswap sends as well as receives so we could change the sentence to:

```suggestion
The core of Helia will be very focused on use as a library: just [js-libp2p], a [datastore], and a [blockstore] that transparently uses [js-bitswap] to load any requested blocks not already in the blockstore from the network and/or to transfer them to network peers.
```

**Comment by BigLep** [comment]
Hi Alex.  I tried looking at this with fresh eyes and will someone not close to Helia get what Helia and why it exists.  

It seems like another key design goal is to break open the "black box" of IPFS and have better visibility into what is happening by allowing a lot more events that can hook in to monitor and debug.

This probably doesn't belong here (maybe FAQ?) but I think we want to callout the things that arne't customizable.  It sounds like the main area is data transfer currently, is that right?  (We're assuming Bitswap and we don't expose configuration of bitswap - is that right?). (We have configurability over Content and Peer routing though as those are controlled by js-libp2p.

A laid visual could likely help here showing the relationship between the various components
* helia
* libp2p node
* datastore
* blockstore
* bitswap
* other helia modules
* user application (consumer of helia)
I don't exactly know how the picture should look  but it seems a bit surprising not to have any architecture diagrams anywhere in the repo.  It will also be useful for those who are more visual in their understanding.
(Understood visual is more work.  That can be a separate item.  I expect it will be useful for IPFS Thing presentation.  Let me know if it would be helpful to whiteboard together.)

**Comment by BigLep** [comment]
Feel free to merge after you pick through the comments that make sense to apply now and backlog the others.

File: MANIFESTO.md [hunk 2]
```diff
@@ -23,27 +22,32 @@ For example:
...
+
+- [`@helia/unixfs`](https://github.com/ipfs/helia-unixfs) - for performing [UnixFS] operations
 
-> **Note**
->
-> An implementation of unixFS compatible with helia can be installed from npm [`@helia/unixfs`](https://github.com/ipfs/helia-unixfs).
+We hope there will be more soon!
 
-## JavaScript first
+## 🥇 JavaScript first
```
[the remark this answers is not in the document]
Comment (achingbrain) [reply MANIFESTO.md:50]: Sounds good

**Pull request merged**